docs/tables/aws_clb_access_log/index.md

```diff
@@ -203,7 +207,7 @@ partition "aws_clb_access_log" "my_logs_regions" {
   source "aws_s3_bucket" {
     connection  = connection.aws.logging_account
     bucket      = "aws-clb-logs-bucket"
-    file_layout = "AWSLogs/(%{DATA:org_id}/)?%{NUMBER:account_id}/elasticloadbalancing/(us-east-1|us-east-2)/%{YEAR:year}/%{MONTHNUM:month}/%{MONTHDAY:day}/%{DATA}.log"
+    file_layout = "AWSLogs/(%{DATA:org_id}/)?%{NUMBER:account_id}/elasticloadbalancing/(us-east-1|us-east-2)/%{YEAR:year}/%{MONTHNUM:month}/%{MONTHDAY:day}/%{NUMBER:account_id}_elasticloadbalancing_%{DATA:region}_[^_]+_[^_]+_[^_]+_[^.]+.log"
```

Comment:
Again, `account_id` is declared twice and is this `[^_]+_[^_]+_[^_]+_[^.]+` required `%{DATA}` would suffice?